[DOC-387] Clarify Storage Execution Times in EXPLAIN (ANALYZE, DIST) (#22768)
yugabyte/yugabyte-db: docs/content/stable/explore/query-1-performance/explain-analyze.md

- **Timings**: At the end of the plan tree, YugabyteDB will add multiple time taken metrics when the [DIST](../../../api/ysql/the-sql-language/statements/perf_explain/#dist) option is specified. These are aggregate times across all plan nodes. Some of them are,
    - _Planning Time_: The time taken in milliseconds for the [query planner](../../../architecture/query-layer/#planner) to design the plan tree. Usually this value is higher the first time a query is executed. During subsequent runs, this time will be low as the plans are cached and re-used.
    - _Execution Time_: The total time taken in milliseconds for the query to execute. This includes all table/index read/write times.

>     - _Storage Read Execution Time_: The total time spent by the YSQL query layer in waiting to receive data from the DocDB storage layer. This is measured as the sum of wait times for individual RPCs to read tables and indexes. The YSQL query layer often pipelines requests to read data from the storage layer, and so the _Storage Read Execution Time_ is often lower than the sum of round-trips times to read tables and indexes.

Reviewer: `This is measured as the sum of wait times for individual RPCs to read tables and indexes. 
`
This line seems redundant 

The YSQL query layer often pipelines requests to read data from the storage layer, and so the _Storage Read Execution Time_ is often lower than the sum of round-trips times to read tables and indexes.

**We need to expand on this : Not very clear for someone which may not have enough context. Taking a attempt to fill bit more context**
 
PostgreSQL after issuing the read request might perform other work thus not start immediately waiting on the response from the StorageLayer. Once PostgreSQL has completed the required work then it starts waiting on response on Storage layer. Hence the total time spent in waiting for storage layer is less then the actual time taken by Storage layer to fulfill a request.

>     - _Storage Read Execution Time_: The total time spent by the YSQL query layer in waiting to receive data from the DocDB storage layer. This is measured as the sum of wait times for individual RPCs to read tables and indexes. The YSQL query layer often pipelines requests to read data from the storage layer, and so the _Storage Read Execution Time_ is often lower than the sum of round-trips times to read tables and indexes.

Reviewer 2: ```suggestion
    - _Storage Read Execution Time_: The total time spent by the YSQL query layer waiting to receive data from the DocDB storage layer. This is measured as the sum of wait times for individual RPCs to read tables and indexes. Because the YSQL query layer often pipelines requests to read data from the storage layer, the _Storage Read Execution Time_ is often lower than the sum of the times for round-trips to read tables and indexes.
```

>     - _Storage Write Execution Time_: The sum of all round-trip times taken to flush buffered write requests to tables and indexes.

Reviewer: This we can leave as previously  or we need to add explanation for what is buffered write implies
